x-pack/plugins/observability_solution/observability/public/pages/slo_details/components/header_control.tsx

```diff
@@ -111,6 +115,9 @@ export function HeaderControl({ isLoading, slo }: Props) {
     (url: string) => setTimeout(() => navigateToUrl(url)),
     [navigateToUrl]
   );
+  const isRemote = !!slo?.remoteName;
+
+  const canEditRemote = Boolean(isRemote && slo.kibanaUrl);
```

Comment:
@grabowskit I think this is a usability/product question... should we assume the user has "edit" permissions and just send them to the remote and let them deal with the fallout if they don't. OR since we don't know if they have edit permissions or not, just disable it?

Reply:
I would enable edit. If they don't have access they will get an error and have to ask for permissions. Usability over caution. :)